### DISCUSSION TITLE
How to import several dictionaries from one folder?

### DISCUSSION THREAD

**Discussion by homocomputeris** [arianneorpilla/jidoujisho#297] [Q&A] [answered]
I use GoldenDict dictionaries. Basically, GD prefers dictionaries arranged like this: GD > language (e.g. `jpn`) > dictionary name (e.g. `kankyusha`, `wiktionary`) > dictionary files. I store them in the same way on Android devices.

How can all dictionaries recursively from, say, `jpn` folder?

**Comment by arianneorpilla** [comment, marked as answer]
You can't. You will have to pick multiple dictionary files in the same folder by selecting those multiple files.
